src/components/ThirdGameRow/ThirdGameRow.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Timer from "../Timer/Timer";
+import VictoryView from "../VictoryView/VictoryView";
+import LossView from "../LossView/LossView";
+import {GameState} from "../../services/CommonTypes";
+
+interface Props {
+    gameState: GameState,
+    timerCount: number,
+    restartGame: () => void
+}
+
+const ThirdGameRow: React.FC<Props> = (props: Props) => {
+    const isGameInitialized = (): boolean => {
+        return props.gameState !== GameState.NOT_INITIALIZED
+    }
+    const isGameInProgress = (): boolean => {
+        return props.gameState === GameState.IN_PROGRESS
+    }
+    const isGameWon = (): boolean => {
+        return props.gameState === GameState.WON
+    }
+    return (
+        <>
+            {isGameInitialized() ? isGameInProgress() ? <Timer timer={props.timerCount}/> :
```

Comment:
For the future: for implementing such state machine-like games like yours, a `react-router` would be really good :)